src/resources/js/components/CreatePresentation.vue

```diff
@@ -0,0 +1,121 @@
+<template>
+  <v-container>
+    <v-alert
+    v-model="message.show"
+    :class="message.type"
+    dismissible
+    >
+      {{ message.content }}
+    </v-alert>
+
+    <h2>Creation de presentation</h2>
+    <v-form
+      ref="form"
+      v-model="valid"
+      lazy-validation
+    >
+      <v-text-field
+        v-model="dataForm.title"
+        :counter="20"
+        :rules="titleRules"
+        label="Titre"
+        required
+      />
+
+      <v-datetime-picker
+        label="Date et Heure"
+        v-model="dataForm.date"
+        :rules="dateRules"
+        clearText="annuler"
+      >
+        <template slot="dateIcon">
+          <v-icon>mdi-calendar</v-icon>
+        </template>
+        <template slot="timeIcon">
+          <v-icon>mdi-clock</v-icon>
+        </template>
+      </v-datetime-picker>
+
+      <v-btn
+        :disabled="!valid"
+        color="success"
+        class="mr-4"
+        @click="createPresentation()"
+      >
+        Créer la présentation
+      </v-btn>
+
+    </v-form>
+  </v-container>
+</template>
+
+<script>
+export default {
+  data: () => ({
+    valid: true,
+    titleRules: [
+      (v) => !!v || 'Vous devez renseigner un titre',
```

Comment:
Is the `!!` normal?

Reply:
it's for force the conversion to boolean value